pyproject.toml

```diff
@@ -34,6 +34,23 @@ profile = "black"
 combine_as_imports = true
 ensure_newline_before_comments = false
 
+[tool.pylint.messages_control]
+disable = [
+    "missing-function-docstring",
+    "missing-module-docstring",
+    "consider-using-f-string",
+    "protected-access",
+    "too-many-branches",
+    "super-with-arguments",
+    "redefined-outer-name",
+    "broad-exception-caught",
+    "no-else-break",
+    "no-else-return",
+    "invalid-name",
+    "import-error",
+    "unnecessary-pass",
+    "unspecified-encoding",
```

Comment:
@bernhardkaindl are these ok to disable?

Reply:
> @bernhardkaindl are these safe to disable?

May I have your further feedback to push this review orward?

Reply:
Hi @psafont , as @bernhardkaindl 's previous perspective (https://github.com/xapi-project/xen-api/pull/5296#discussion_r1424543694), he thought those warnings are harmless and even far less of a fixed "problem" that would belong to a Python3 PR.
So it is fine to disable them.

Reply:
`"unspecified-encoding"` is not safe to ignore blindly for all files.

The reason is that files can contain UTF-8 characters or even non UTF-8 bytes `> ord(127)`, that are not valid UTF-8 and this caused crashing the auto-cert-kit Xapi plugin last year:

```py
LC_ALL=C python3.6 -c 'open("/usr/share/hwdata/pci.ids").read()'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/lib64/python3.6/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 90372: ordinal not in range(128)
```

Reply:
> If enabling these rule, we have to add dozens more lines of changes, and even refactor the logic for some rules, all of which are risky.

Hi @acefei Would you please create a new ticket to track this kind of changes? It would be a good chance to make these potential issues more clear in feature branch.